app/screens/home/bottomTabs/Articles.tsx

```diff
@@ -421,76 +448,169 @@ const Articles = ({ route, navigation }: any): any => {
 
 
   const onFilterArrayChange = (newFilterArray: any): any => {
-
+    console.log('filterarray data', newFilterArray)
     setFilterArray(newFilterArray)
   }
+  const customSortFunction = (a: any, b: any, queryText: any): any => {

```

Comment:
You can explicitly specify types of a and b as string, if they are always expected to be strings.